examples/interdomain/dns/README.md

```diff
@@ -72,6 +72,11 @@ data:
         loop
         reload 5s
     }
+    # Azure reverse DNS zone
+    internal.cloudapp.net:53 {
+      errors
+      forward . 168.63.129.16
```

Comment:
How are we getting 168.63.129.16?

Reply:
It is predefined IP address for Azure DNS server:
`Access to Azure's recursive resolvers is provided via the virtual IP 168.63.129.16.`
https://github.com/uglide/azure-content/blob/master/articles/virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances.md#name-resolution-using-your-own-dns-server